### DISCUSSION TITLE
Is it possible to migrate a space to a new ENS?

### DISCUSSION THREAD

**Discussion by jzdhuang** [snapshot-labs/snapshot#2876] [Q&A] [answered]
Hi,
If we have an existing space but hope to rename it under another ENS, is that possible?

**Comment by metrox-eth** [comment, marked as answer]
Hey, this is possible but require a manual migration from our side.
Here is how to initiate the migration:
Create a proposal on the source space as well as on the target space with the title "Migrate this space" from the respective controllers of the spaces (the Ethereum address that is registered in the snapshot ENS text record.
One last step, get in touch with @space-lead on our [Discord server](https://discord.snapshot.org/)
Please allow a couple of days for the workflow to be processed.
